### PR TITLE
fix: dark theme in 4.0

### PR DESCRIPTION
**Pull request by ycjcl868**
### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / document update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |           |
| 🇨🇳 Chinese |           |

### ☑️ Self Check before Merge

- [ ] Doc is updated/provided or not needed
- [ ] Demo is updated/provided or not needed
- [ ] TypeScript definition is updated/provided or not needed
- [ ] Changelog is provided or not needed

-----
[View rendered components/button/index.en-US.md](https://github.com/ant-design/ant-design/blob/fix-ui-dark/components/button/index.en-US.md)
[View rendered components/button/index.zh-CN.md](https://github.com/ant-design/ant-design/blob/fix-ui-dark/components/button/index.zh-CN.md)
[View rendered components/config-provider/demo/direction.md](https://github.com/ant-design/ant-design/blob/fix-ui-dark/components/config-provider/demo/direction.md)
[View rendered components/table/index.zh-CN.md](https://github.com/ant-design/ant-design/blob/fix-ui-dark/components/table/index.zh-CN.md)
[View rendered components/transfer/index.zh-CN.md](https://github.com/ant-design/ant-design/blob/fix-ui-dark/components/transfer/index.zh-CN.md)

### CODE REVIEW

File: docs/spec/download.en-US.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ title: Resources
 Please find below some of the design resources and tools about Ant Design that we consider valuable. More of this is still being collected.
 
 <div class="resource-cards">
-  <a target="_blank" href="https://github.com/ant-design/ant-design/releases/download/resource/Ant.Design.Components.3.12.0.sketch" class="resource-card">
+  <a target="_blank" href="https://github.com/ant-design/ant-design/releases/download/4.0.0-beta.1/Ant.Design.Components.4.0.Beta.sketch" class="resource-card">
```
Comment (afc163) [inline docs/spec/download.en-US.md]: 404？

Comment (ycjcl868) [reply docs/spec/download.en-US.md]: 我找个 4.0 tag 贴下

Comment (ycjcl868) [reply docs/spec/download.en-US.md]: 可以下了

File: docs/spec/download.zh-CN.md [hunk 2]
```diff
@@ -9,7 +9,7 @@ title: 设计资源
 - 新版本 Sketch Symbols 组件反馈地址：[🦜 反馈地址](https://www.yuque.com/kitchen/topics/51)
 
 <div class="resource-cards">
-  <a target="_blank" href="https://github.com/ant-design/ant-design/releases/download/resource/Ant.Design.Components.3.12.0.sketch" class="resource-card">
+  <a target="_blank" href="https://github.com/ant-design/ant-design/releases/download/4.0.0-beta.1/Ant.Design.Components.4.0.Beta.sketch" class="resource-card">
```
Comment (afc163) [inline docs/spec/download.zh-CN.md]: 为啥不统一上传到 https://github.com/ant-design/ant-design/releases/resource ？

Comment (ycjcl868) [reply docs/spec/download.zh-CN.md]: ok

File: site/theme/static/index.less [hunk 3]
```diff
@@ -24,3 +24,4 @@
 @import './nprogress';
 @import './contributors';
 @import './dark.less';
+@import './rtl.less';
```
Comment (ycjcl868) [inline site/theme/static/index.less]: use `rtl.less` in site, @saeedrahimi.

**Pull request merged**